# TrajectoryMsg.msg
geometry_msgs/Point[8] trajectory_points
